Update FCERM finder to filter on date of completion

brucebolt opened the pull request:
The FCERM (Flood and Coastal Erosion Risk Management) finder currently uses the published date as a filter facet. This changes that to the date of completion, which is more useful to the users of this finder.

Make live by running this rake task:
```
publishing_api:publish_finder[flood_and_coastal_erosion_risk_management_research_reports]
```

Trello card: https://trello.com/c/yNBVwh6S

brucebolt committed:
Update FCERM finder to filter on date of completion

The FCERM (Flood and Coastal Erosion Risk Management) finder currently
uses the published date as a filter facet. This changes that to the date
of completion, which is more useful to the users of this finder.

thomasleese reviewed (commented):
Changes look good, I can't remember if this is in the card, but I wonder if should change the `start_date` name to `Date of start` as well so they're consistent when they show up in the result metadata?

brucebolt committed:
Update label for start date in FCERM finder

Changes the label for the FCERM (Flood and Coastal Erosion Risk
Management) finder's 'start date' to be consistent with the date of
completion.
